app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    total_rating = sum(player.get_rating() for player in players)
+    return total_rating
```

Comment:
```suggestion
    return sum(player.get_rating() for player in players)
```
no need to create variable in this case